src/engine/enginebuffer.cpp

```diff
@@ -1284,6 +1296,14 @@ bool EngineBuffer::isTrackLoaded() {
     return false;
 }
 
+bool EngineBuffer::isSeekQueued(ControlValueAtomic<double> &seekPos) {
```

Comment:
we use only const references in Mixxx. I think we can use here a "double* pSeekPosition" as second return value.